dockstore-webservice/src/test/java/io/dockstore/webservice/resources/UserResourceTest.java

```diff
@@ -0,0 +1,26 @@
+package io.dockstore.webservice.resources;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import io.dockstore.webservice.CustomWebApplicationException;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class UserResourceTest {
+
+    @Test
+    public void usernameRestrictionsTest() {
+        List<String> invalidUsernames = new ArrayList<>(List.of("dockstore", "DOCKSTORE", "thisIsDoCkStOrE", "admin", "aaaaadminbbb", "curator",
```

Comment:
Ideally you would have some valid usernames as well, to make sure your code just doesn't always reject everything. Not that it does, and it's pretty straightforward. Just for future reference.